doc/ref_model/chapters/chapter04.md

```diff
@@ -94,7 +94,7 @@ This section describes a set of explicit NFVI capabilities and performance measu
 | e.nfvi.cap.009 | Crypto Acceleration | Yes/No | Crypto Acceleration |
 | e.nfvi.cap.010 | Transcoding Acceleration | Yes/No | Transcoding Acceleration |
 | e.nfvi.cap.011 | Programmable Acceleration | Yes/No | Programmable Acceleration |
-| e.nfvi.cap.012 | Enhanced Cache Management* | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded |
+| e.nfvi.cap.012 | Enhanced Cache Management* | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded </br> **Note:** This capability is not applicable to containers or Kubernetes based infrastructures.|
```

Comment:
@CsatariGergely Please clarify why K8s infra wouldn't use a CPU's cache, and by extension, its cache management. Thx, -M
P.S. Does this need to be in RM? Prefer to see in the RA's compliance/traceability matrix.

Reply:
@CsatariGergely Hi Gergely, What was the change you made here? It looks like the change was backed out, but hard to tell since I'm still figuring out this area of github. Thx, -Mark

Reply:
Ok, looks like it was indeed backed out. Don't forget to back it out down below in the mapping table (4-24), so it's consistent. Thanks!

Reply:
@markshostak yes, I've removed the comments. I did not change anything in the mapping table related to this.